peachjam/views/article.py

```diff
@@ -0,0 +1,36 @@
+from django.shortcuts import get_object_or_404
+from django.views.generic import DetailView, ListView
+
+from peachjam.models import Article, UserProfile
+
+
+class ArticleListView(ListView):
+    model = Article
+    template_name = "peachjam/article_list.html"
+    context_object_name = "articles"
+    navbar_link = "articles"
+    ordering = ["-id"]
+    paginate_by = 5
+
+
+class ArticleDetailView(DetailView):
+    model = Article
```

Comment:
Queryset should filter on published articles, otherwise draft articles will be visible.